docs/Facilities.md

```diff
@@ -59,3 +120,50 @@ contxtSdk.facilities.getAll()
   .then((facilities) => console.log(facilities));
   .catch((err) => console.log(err));
 ```
+<a name="Facilities+getAllByOrganizationId"></a>
+
+### contxtSdk.facilities.getAllByOrganizationId(organizationId) ⇒ <code>Promise</code>
+Gets a list of all facilities that belong to a particular organization
+
+API Endpoint: '/organizations/:organizationId/facilities'
+Method: GET
+
+**Kind**: instance method of [<code>Facilities</code>](#Facilities)  
+**Fulfill**: <code>Facility[]</code> Information about all facilities  
+**Reject**: <code>Error</code>  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| organizationId | <code>string</code> | UUID corresponding with an organization |
+
+**Example**  
+```js
+contxtSdk.facilities.getAllByOrganizationId(25)
+  .then((facilities) => console.log(facilities));
+  .catch((err) => console.log(err));
+```
+<a name="Facilities+update"></a>
+
+### contxtSdk.facilities.update(facilityId, update) ⇒ <code>Promise</code>
+Updates a facility's specifics
+
+API Endpoint: '/facilities/:facilityId'
+Method: PUT
+
+**Kind**: instance method of [<code>Facilities</code>](#Facilities)  
+**Fulfill**: <code>undefined</code>  
+**Reject**: <code>Error</code>  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| facilityId | <code>number</code> | The id of the facility to update |
+| update | <code>Object</code> | An object containing the updated data for the facility |
```

Comment:
Do you want to list out all of the possible things that could be updated?

Reply:
This may be listed out elsewhere in the documentation; I'm not quite sure

Reply:
Oh yeah, good idea. I did it for create, but not update